scripts/backend_tests.py

```diff
@@ -307,6 +307,9 @@ def _get_test_target_classes(path):
 
 def main():
     """Run the tests."""
+    # Change default encoding of Python to 'utf-8'.
+    reload(sys)  # pylint: disable=reload-builtin
```

Comment:
This is odd -- why do you need to reload in order to set the default encoding? What happens if you remove this statement?

Reply:
I tried printing unicode characters in the test file but that gives an unicode error as all the other strings are printed in  bytes. In python2, the environment is in ascii while in python3, the environment is in utf-8. That's why I want to change the environment so that all the characters are properly printed.

Reply:
We've been working in a python 2 environment for a long time. How has unicode printing in this environment worked historically?

Reply:
Removed